state-chain/pallets/cf-witnesser/src/lib.rs

```diff
@@ -376,10 +376,10 @@ pub mod pallet {
 		/// - [DuplicateWitness](Error::DuplicateWitness)
 		#[allow(clippy::boxed_local)]
 		#[pallet::call_index(0)]
-		#[pallet::weight(
+		#[pallet::weight((
 			T::WeightInfo::witness_at_epoch().saturating_add(call.get_dispatch_info().weight /
 				T::EpochInfo::authority_count_at_epoch(*epoch_index).unwrap_or(1u32) as u64)
-		)]
+		, DispatchClass::Operational))]
 		pub fn witness_at_epoch(
```

Comment:
force_witness too?

Reply:
Force witness is a governance extrinsic so it's only ever dispatched via goverannce approval.